src/index.js

```diff
@@ -296,6 +296,21 @@ const ScanThng = {
     api.scopes.Operator.prototype.stopStream = stopStream;
     api.scopes.Operator.prototype.scan = scan;
   },
+  /**
+   * Put image on canvas, convert it and export as data URL.
+   *
+   * @param {*} imageData - Image data.
+   * @param {object} userOptions - User options, if any.
+   * @returns {Promise}
+   */
+  processImage: Media.processImage,
+  /**
+   * Read file selected by user.
+   *
+   * @param {object} file - The file chosen by the user.
+   * @returns {Promise}
+   */
+  readUserFile: Media.readUserFile,
```

Comment:
Suggest `convertToDataUrl` as the exported name, and similarly add docs for the return type.